design/one-pager-stack-relationship-labels.md

```diff
@@ -0,0 +1,165 @@
+# Stack Parent/Child Relationship Label Spec
+
+- Owner: Steven Rathbauer (@rathpc)
+- Reviewers: Crossplane Maintainers
+- Status: Draft
+
+## Proposal
+
+[crossplaneio/crossplane#934](https://github.com/crossplaneio/crossplane/issues/934)
+
+## Problem
+
+There is currently no easy way to get a list of all resources that are owned by a parent resource.
+We can leverage CRD labels to make this easier. As long as we have a standard format for doing so.
+
+Per the proposal, it seems that the closest example of what I am referring to is shown on this page
+https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels within the
```

Comment:
Nitpick: I would normally expect this to be a fancy link, such as:

```
shown on [this page](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels), within the . . . 
```

That way it takes up less space for readers who're looking at the rendered markdown.